src/java.base/share/classes/java/io/ObjectInputFilter.java

```diff
@@ -650,7 +639,7 @@ static ObjectInputFilter rejectUndecidedClass(ObjectInputFilter filter) {
                 try {
                     filter = createFilter(filterString);
                 } catch (RuntimeException re) {
-                    configLog.log(System.Logger.Level.ERROR,
+                    configLog.log(ERROR,
                             "Error configuring filter: {0}", re);
```

Comment:
You should either remove "{0}" from the string, or cast re to `Object` - as there is an overridden log method that takes a Throwable. If you want to display the exception stack trace, just call:

```
configLog.log(ERROR, "Error configuring filter", re);
```

If you don't want to display the stack trace, call:

```
configLog.log(ERROR, "Error configuring filter: {0}", (Object)re);
```

Reply:
Hard to spot that mis-use.
Also observed that if the jdk.serialFilter is ill formed (IAE), it gets logged but should also rethrow
the exception to prevent continuing without an expected filter.